assets/javascripts/test_result.js

```diff
@@ -748,40 +748,61 @@ function renderInvestigationTab(response) {
 
     var tbodyElement = document.createElement('tbody');
     Object.keys(response).forEach(key => {
+        var value = response[key];
+        var type = 'pre';
+
+        //The value could be an object. This allows to extend the functionality
+        //this object will have the attribute type that determines the behavior
+        //The accepted types are:
+        // - link: adds a link reference using an anchor <a>
+        //   additional required attributes:
```

Comment:
I guess it should be additional**ly** here.

Reply:
hm, no as it's an adjective to a simple description, not a complete sentence: "required attributes:" is just specified with an additional adjective to "additional required attributes:"

Reply:
I thought so because it is followed by the adjective "required" (which is followed by the noun "attributes"). When I remember correctly, adjectives are used to describe nouns and adverbs to describe adjectives (regardless whether it is a full sentence or not). Not sure what you mean with "simple description".